service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -105,4 +106,11 @@ public MetaDbSample getMetaDbSampleByRequestAndIgoId(String requestId, String ig
         }
         return requestSamples;
     }
+    
+    @Override
+    public List<SampleMetadata> retrieveSampleMetadataHistory(String igoSampleId) throws Exception {
+        List<SampleMetadata> requestSamples = sampleRepository.retrieveSampleMetadataHistory(igoSampleId);
+        Collections.sort(requestSamples);
```

Comment:
ah, is this what the `Comparable<SampleMetadata>`  is for from above?

Reply:
by default do we know in what order they are sorted in? like ascending or descending by date?

Reply:
Yes it is ascending

Reply:
can you add that as a comment just for future reference? it can be on the same line as the `.sort()`